test/system/src/main/java/io/pravega/test/system/framework/services/PravegaControllerService.java

```diff
@@ -140,7 +140,7 @@ private App createPravegaControllerApp() {
                 setSystemProperty("REST_SERVER_PORT", String.valueOf(REST_PORT)) +
                 setSystemProperty("log.level", "DEBUG") +
                 setSystemProperty("curator-default-session-timeout", String.valueOf(10 * 1000));
-        Map<String, String> map = new HashMap<>();
+        Map<String, Object> map = new HashMap<>();
```

Comment:
There a few changes from `String` to `Object`, is this related to any of the version upgrades?

Reply:
Yes. The type on the curator interface changed.